MobileWallet/Screens/Send/AddRecipient/AddRecipientViewController.swift

```diff
@@ -398,11 +398,14 @@ class AddRecipientViewController: UIViewController, UITextFieldDelegate, Contact
 
         let pasteboardString: String? = UIPasteboard.general.string
 
-        if let text = pasteboardString {
+        if var text = pasteboardString {
             //Try get a pubkey from clipboard text
             do {
-                let pubKeyFromDeeplink = try PublicKey(any: text)
-                clipboardEmojis = pubKeyFromDeeplink.emojis.0
+                if !text.hasPrefix("\(TariSettings.shared.deeplinkURI)://") {
```

Comment:
@SerhiiShovkoplias It's still problematic because the `PublicKey` constructor can function with 3 types of inputs: deep link, public key hex string or emoji id. The issue in #421 is that a digit string literal is considered to be an emoji by iOS.

Now that we can get the set of Tari emojis from FFI, I think we'll only need to change the logic that extracts the emojis and #421 will be fixed. I will post an update here tomorrow once we get the FFI updates merged, then we can have the ultimate solution.